.taskcat.yml

```diff
@@ -7,62 +7,16 @@ project:
   parameters:
     QSS3BucketName: $[taskcat_autobucket]
     QSS3KeyPrefix: "cfn-ecr-aws-soci-index-builder/"
-  regions:
-  - us-east-1
-  - eu-west-1
-  - us-west-1
-  - ap-southeast-1
-  - ap-northeast-1
-  - us-west-2
-  - sa-east-1
-  - ap-southeast-2
-  - eu-central-1
-  - ap-northeast-2
-  - ap-south-1
-  - us-east-2
-  - ca-central-1
-  - eu-west-2
-  - eu-west-3
-  - ap-northeast-3
-  - eu-north-1
-  - ap-east-1
-  - me-south-1
-  - eu-south-1
-  - af-south-1
-  - us-gov-west-1
-  - us-gov-east-1
-  - cn-north-1
-  - cn-northwest-1
-  - me-central-1
 tests:
   can-deploy:
     parameters:
       SociRepositoryImageTagFilters: "*:*"
     regions:
       - us-east-1
-      - eu-west-1
```

Comment:
With the V2 change, it is the responsibility of the caller to configure the taskcat.yml file including the regions that need to be supported. we should add more details about this in the doc